.github/workflows/qa.yaml

```diff
@@ -123,7 +136,7 @@ jobs:
           CGO_CFLAGS: "-O0 -g3 -fno-omit-frame-pointer"
         run: |
           # Use `-dwarflocationlists` to give ASAN a better time to unwind the stack trace
-          go test -C pam -asan -gcflags="-dwarflocationlists=true" ./...
+          go test -asan -gcflags="-dwarflocationlists=true" ./pam/pam_test
```

Comment:
Mh, is it a problem to run also integration tests under asan? They seem to work fine to me.

However, this change is fine, but it would also avoid the check to happen in #125 and #121, so it should be changed to also include `./pam/gdm` at least.

Reply:
We already run them twice with `go test ./...` and with `go test -race ./...`. Since they take quite a while to run, I don't think we want to keep running them this much.

Reply:
I'll add the `./pam/gdm` to the command, though.